policyengine_us/parameters/gov/states/nc/ncdhhs/scca/infant_age_limit.yaml

```diff
@@ -0,0 +1,10 @@
+description: North Carolina categorizes children under below age as Infant group in its Subsidized Child Care Assistance program.
+metadata:
+  reference:
+    - title: Subsidized Child Care Market Rates for Child Care Centers - Effective 10-1-2023
+      href: https://ncchildcare.ncdhhs.gov/Portals/0/documents/pdf/M/Market_Rates_Centers_Eff_10-1.pdf?ver=9w52alSPhmrmo0N9gGVMEw%3d%3d
```

Comment:
Could you remove references if they don't specifically point to the policy parameter which is mentioned in the yaml file

Reply:
Do you want me to remove this reference in other age limit files also, Eg, preschool_age_lower, toddler_age_limit, etc.?

Reply:
Could we fine statutory reference that show each age limit?

Reply:
I found a clear definition only for the infant age group and included the document in the reference. I also added a comment in `two_year_olds_age_limit.yaml` explaining how the two-year-olds group is categorized.